pkg/koordlet/resmanager/cgroup_reconcile.go

```diff
@@ -36,6 +37,8 @@ import (
 	"github.com/koordinator-sh/koordinator/pkg/util/sloconfig"
 )
 
+var pageSize = int64(syscall.Getpagesize())
```

Comment:
It is recommended to move this variable under the pkg koordlet/util/system.

Reply:
done